LICENSE

```diff
@@ -0,0 +1,4 @@
+Copyright (c) 2015 Smooch Technologies Inc. All rights reserved.
```

Comment:
Should that be 2018? 🤔

Reply:
I think I copied this from another similar repo that already had a license. Will change.